compose.yaml

```diff
@@ -13,9 +13,11 @@ services:
       # - DATABASE_URL=postgres://postgres:postgres@postgres:5432/formulosity?sslmode=disable
       - DATABASE_URL=/root/sqlite3/formulosity.db
       - SURVEYS_DIR=/root/surveys
+      - UPLOADS_DIR=/root/uploads
     volumes:
       - ./api/surveys:/root/surveys
       - ./api/sqlite3:/root/sqlite3
+      - uploadvolume:/root/uploads
```

Comment:
let's keep the same format as other volumes, for example `./api/uploads` and add it to gitignore.